packages/lit-html/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es2020",
+    "target": "es2019",
```

Comment:
This is because rollup-plugin-filesize doesn't support ES2020. It increases the bundle size by a few 10s of bytes, but I think by release we'll be able to change back to ES2020 or ES2021 even (Safari 14 beta is the last major to add logical assignment).